qa/L0_response_cache/test.sh

```diff
@@ -42,6 +43,25 @@ if [ $? -ne 0 ]; then
 fi
 set -e
 
+# SERVER TESTS
+mkdir -p "${PWD}/models/decoupled_cache/1"
+
+# Assert that server fails to start for a "decoupled" model with response
+# cache enabled
+SERVER=/opt/tritonserver/bin/tritonserver
+SERVER_ARGS="--model-repository=${PWD}/models --response-cache-byte-size=8192"
+SERVER_LOG="./inference_server.log"
+source ../common/util.sh
+run_server
+if [ "$SERVER_PID" != "0" ]; then
+    echo -e "\n***\n*** Failed: $SERVER started successfully when it was expected to fail\n***"
+    cat $SERVER_LOG
+    RET=1
+
+    kill $SERVER_PID
+    wait $SERVER_PID
+fi
```

Comment:
Add a simple check to prove the server failed with "Response Cache does not currently support the 'decoupled' model transaction policy" and not some other error